src/assets/styles/_animation.scss

```diff
@@ -41,18 +41,18 @@
 }
 
 :root {
-  --calcite-internal-animation-timing-factor: var(--calcite-animation-timing-factor, 1);
-  --calcite-internal-animation-timing: calc(150ms * var(--calcite-internal-animation-timing-factor));
-  --calcite-internal-animation-timing-quick: calc(100ms * var(--calcite-internal-animation-timing-factor));
```

Comment:
Can we use 'fast' here? Nitpick but we already use that terminology on `alert`.

Reply:
change the name to `fast` . Good catch @macandcheese